### ISSUE TITLE
Populate EN Registry - Part 2

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/registry#99]

**Comment by jordanpadams** [comment]
📆 February Status: Task start delayed. No impact on build. Will not be tested by I&T.

**Comment by jordanpadams** [comment]
📆 March status: Deferred to B14.0 due to other high priority bug fixes. Review with SWG per https://github.com/NASA-PDS/pds-swg/issues/22. No impact on I&T

**Comment by jordanpadams** [comment]
📆 June status: Delayed to end of build. This is mostly an operations task so it will not impact B14.0 delivery.

**Comment by jordanpadams** [comment]
📆 July status: Further delayed. May get pushed to beginning of B14.1 when we move to multi-tenancy. This is mostly an operations task so it will not impact B14.0 delivery.

**Comment by jordanpadams** [comment]
📆 August status: Deferring to B14.1. Need design in place before we can migrate. Initial scripts completed per https://github.com/NASA-PDS/registry/issues/168

**Comment by jordanpadams** [comment]
descoping from B14.1. Too much other more important work
